src/olm/account/mod.rs

```diff
@@ -896,8 +896,8 @@ mod dehydrated_device {
             let one_time_keys: Vec<_> = account
                 .one_time_keys
                 .secret_keys()
-                .iter()
-                .map(|(_key_id, secret_key)| OneTimeKey { private_key: secret_key.to_bytes() })
+                .values()
```

Comment:
Is this related to the olm-sys change?

Reply:
It's not, the commit tells you that it's a new clippy warning that is being fixed.

Reply:
Sorry, I assumed the clippy warnings were related to the changes made in the first commit.